### PR TITLE
Gas physics docs

### PR DESCRIPTION
**Pull request by jimrojerTNO**

### CODE REVIEW

File: docs/theory/gas_physics.rst [hunk 1]
```diff
@@ -3,4 +3,218 @@
...
+
+Asset Physics
+-------------
+
+Source
+~~~~~~
+
+The source adds gas mass to the network.
+The volumetric- and mass flow balance is given by :eq:`eq:Q_balance` and :eq:`eq:m_balance` where
+:math:`\dot{V}^a_{consumed}` :math:`\dot{\m}^a_{consumed}` is equal to the (negative) value of the
```
Comment (KobusVanRooyen) [inline docs/theory/gas_physics.rst:158]: - [ ] Should this be volumetric flow rate / mass flow rate?
- [ ] Should the mass flow rate symbol be red?

File: docs/theory/gas_physics.rst [hunk 2]
```diff
@@ -3,4 +3,218 @@
...
+
+    H^a_{pump} = H^a_{out} \;\; \forall a \in A_{prod}.
+
+Demand
+~~~~~~
+
+A demand extracts gas from the network, defined by :eq:`eq:Q_balance` and :eq:`eq:m_balance` where
+:math:`\dot{V}^a_{consumed}` and :math:`\dot{m}^a_{consumed}` is the consumed gas from the network.
+
+Similar to the source, no constraints are required at the in-going supply side of the demand, and
```
Comment (KobusVanRooyen) [inline docs/theory/gas_physics.rst:174]: Should this be deleted?

**Comment by KobusVanRooyen** [comment]
I was wondering if we should not refer to heat physics for the section that are esactly the same. This would reduce maintenance in the future. What do you think @jimrojerTNO

File: docs/theory/gas_physics.rst (hunk 1 above)
Comment (jimrojerTNO) [reply docs/theory/gas_physics.rst:158]: Yes for the first one

for the second point, I personally don't think so, but if your prefer to use red please do so

File: docs/theory/gas_physics.rst (hunk 2 above)
Comment (jimrojerTNO) [reply docs/theory/gas_physics.rst:174]: yes should be deleted

File: docs/theory/gas_physics.rst [hunk 3]
```diff
@@ -3,4 +3,217 @@
 Gas Physics
 ===========
 
-Coming soon...
+MESIDO models the volumetric flow (:math:`\dot{V}`), the mass flow (:math:`\dot{m}`), the head
```
[the remark this answers is not in the document]
Comment (jimrojerTNO) [reply docs/theory/gas_physics.rst:6]: indeed

**Comment by jimrojerTNO** [comment]
> I was wondering if we should not refer to heat physics for the section that are esactly the same. This would reduce maintenance in the future. What do you think @jimrojerTNO

I think we can only do this for the logic of the head losses and hydraulic power. I do think it is a good idea. But let's maybe do this in the next iteration.

File: docs/theory/gas_physics.rst (hunk 2 above)
Comment (KobusVanRooyen) [reply docs/theory/gas_physics.rst:174]: deleted

File: docs/theory/gas_physics.rst (hunk 1 above)
Comment (KobusVanRooyen) [reply docs/theory/gas_physics.rst:158]: Fixed

**Review by KobusVanRooyen — APPROVED** [review]

**Pull request merged**